tacacs_plus/authentication.py

```diff
@@ -42,8 +44,9 @@ def packed(self):
         # B = unsigned char
         # s = char[]
         username = six.b(self.username)
+        rem_addr = six.b(self.rem_addr)
+        port = six.b(self.port)
```

Comment:
It might be good to update the `port_len: 0` and `rem_addr_len: 0` lines in `__str__` below.